perception/elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -141,7 +141,7 @@ void ElevationMapLoaderNode::publish()
     try {
       is_bag_loaded = grid_map::GridMapRosConverter::loadFromBag(
         *data_manager_.elevation_map_path_, "elevation_map", elevation_map_);
-    } catch (rosbag2_storage_plugins::SqliteException & e) {
+    } catch (const std::runtime_error & e) {
       is_bag_loaded = false;
```

Comment:
Please add warning or error messages for e.what()

Reply:
Thanks!
I added this in  https://github.com/autowarefoundation/autoware.universe/pull/4187/commits/21507d346f76e20fe87ff06c82c922784d8a9ce6